### DISCUSSION TITLE
SDXL Model Loading Problem

### DISCUSSION THREAD

**Discussion by rafiislambd** [Comfy-Org/ComfyUI#3857] [Q&A]
My gpu rtx 3060 12gb but first time model loading 10/15 minute. how to fix this?

**Comment by mcmonkey4eva** [comment]
model loading times are based on data drive speed, not GPU

**Reply by rafiislambd** [reply]
oh.thanks

**Comment by ltdrdata** [comment]
You need to store model into SSD instead of HDD.

**Reply by Robson1970** [reply]
https://github.com/comfyanonymous/ComfyUI/issues/1992

Open utils.py with the editor, search for  **sd = safetensors.torch.load_file(ckpt, device=device.type)** 
replace the line to **sd = safetensors.torch.load(open(ckpt, 'rb').read())**
